packages/rest/src/keys.ts

```diff
@@ -5,7 +5,12 @@
 
 import {CoreBindings} from '@loopback/core';
 import {BindingKey, Context} from '@loopback/context';
-import {OpenApiSpec} from '@loopback/openapi-v3-types';
+
+/**
+ * See https://github.com/Microsoft/TypeScript/issues/26985
+ */
+// import {OpenApiSpec} from '@loopback/openapi-v3-types';
+import {OpenAPIObject} from 'openapi3-ts';
```

Comment:
Can you please rename `OpenAPIObject` to `OpenApiSpec` to avoid changing the places using this type below? We don't want to rename them from `OpenApiSpec` to `OpenAPIObject` now and then rename them back to `OpenApiSpec` when the typescript bug is fixed.

```ts
import {OpenAPIObject as OpenApiSpec} from 'openapi3-ts';
```

Reply:
Good idea.